src/main/java/io/aiven/kafka/connect/opensearch/BulkProcessor.java

```diff
@@ -499,7 +573,8 @@ private static ConnectException toConnectException(final Throwable t) {
     public enum BehaviorOnMalformedDoc {
         IGNORE,
         WARN,
-        FAIL;
+        FAIL,
+        REPORT;
```

Comment:
We should update the `OpensearchSinkConnectorConfig.BEHAVIOR_ON_MALFORMED_DOCS_DOC` docstring to explain what this new value will do.